### PR TITLE
Adding pr template and contributing guidlines for new contributors

### PR DESCRIPTION
**Pull request by MandyMeindersma**
@Jhemming93

If this was here would your life been easier? Are there errors and cases I missed? Is this written well or should I reword anything?

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+### Do we follow any naming conventions for branches or anything else?
+
+-   Not really. We aren't picky about branch or commit names. We might be picky about variable names, just that you picked one that would make sense to a lot of people. We are very picky about our linking though! Run the linter!
+
+# Windows
+
+If you have a windows machine there might be a couple issues that come up. Some that we know so far:
+
+-   You might have to remove the single quotes in the `link-check` [file list](https://github.com/devedmonton/DES-Website/blob/main/package.json#L13). Just don't commit this change when you make your pr.
```
Comment (Jhemming93) [inline .github/CONTRIBUTING.md:40]: ned to change "link-check" to "lint-check"

File: .github/CONTRIBUTING.md
Comment (Jhemming93) [inline .github/CONTRIBUTING.md:1]: the only thing i might bring up is to format the code they edit or add with prettier, which will save them alot of headache with the linter

**Review by Jhemming93 — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md
Comment (MandyMeindersma) [reply .github/CONTRIBUTING.md:1]: added

**Review by Jhemming93 — APPROVED** [review]

**Comment by MandyMeindersma** [comment]
@allcontributors add @Jhemming93 for review

**Pull request merged**